assets/messages.yml

```diff
@@ -418,11 +373,11 @@ messages:
       category: duplicate
       message: |-
         *Thank you for your report!*
-        We're tracking this issue in *%s%*, so this ticket is being resolved and linked as a *duplicate*.
+        We're tracking this issue in *[%s%|https://bugs.mojang.com/browse/%s%]*, so this ticket is being resolved and linked as a *duplicate*.
```

Comment:
Could link to portal here.
https://report.bugs.mojang.com/servicedesk/customer/portal/%project_id%/%s%

Reply:
Prefer the public view ones since these types of comments are visible everywhere (also the format you provided doesn't work, we need to add a separate field (something like `project_numeric_id`?): https://report.bugs.mojang.com/servicedesk/customer/portal/2/MC-1 works, https://report.bugs.mojang.com/servicedesk/customer/portal/MC/MC-1 (which is what your format would provide) does not)

Reply:
Also in the portal the resolution for older reports is not visible (i. e. https://report.bugs.mojang.com/servicedesk/customer/portal/2/MC-94498 which was resolved as WAI, but there is no such mention in the comments or anywhere in the portal)

Reply:
That format does seem to work. I've been using it on my messages and it gives results that do lead to the issue.
Link from my template: https://report.bugs.mojang.com/servicedesk/customer/portal/2/MC-102223
Normal portal link: https://report.bugs.mojang.com/servicedesk/customer/portal/2/MC-102223 

I am happy to link to the public view as it's displaying more helpful info currently.

Reply:
Oh, well it seems like you modified the project_id values to the ones on the service desk.